layers/+misc/nav-flash/packages.el

```diff
@@ -0,0 +1,70 @@
+;;; packages.el --- nav-flash layer packages file for Spacemacs.
```

Comment:
Need to update header
https://github.com/syl20bnr/spacemacs/blob/develop/CONTRIBUTING.org#file-header